api/src/resources/comment.rs

```diff
@@ -170,16 +170,63 @@ pub struct CommentsIterPage {
 }
 
 #[derive(Debug, Clone, Serialize)]
-pub(crate) struct PutCommentsRequest<'request> {
-    pub comments: &'request [NewComment],
+pub(crate) struct PutCommentsRequest {
+    pub comments: Vec<NewComment>,
 }
 
-#[derive(Debug, Clone, Deserialize)]
+impl SplittableRequest for PutCommentsRequest {
+    fn split(&self) -> Vec<Self>
+    where
+        Self: Sized,
+    {
+        self.comments
+            .clone()
+            .into_iter()
+            .map(|comment| Self {
+                comments: [comment].to_vec(),
+            })
+            .collect()
+    }
+
+    fn count(&self) -> usize {
+        self.comments.len()
+    }
+}
+
+#[derive(Default, Debug, Clone, Deserialize)]
 pub struct PutCommentsResponse;
 
+impl ReducableResponse for PutCommentsResponse {
+    fn empty() -> Self {
+        Self {}
+    }
+    fn merge(_a: Self, _b: Self) -> Self {
+        Self {}
+    }
```

Comment:
You can supply a [default implementation](https://doc.rust-lang.org/book/ch10-02-traits.html#default-implementations) for trait methods in the trait itself that will be used if there's no implementation for the type. This should allow you to avoid repeating these trivial implementations (you'll need a `Default` [trait bound](https://doc.rust-lang.org/book/ch19-03-advanced-traits.html#using-supertraits-to-require-one-traits-functionality-within-another-trait)). 
```

Reply:
I chose not to use the `Default` trait initially as it doesn't make sense to have `Default` for these responses unless you're using the `ReducableResponse` trait. Don't have strong opinions here though

Reply:
In that case you can make your own version of `Default` called `ReducibleDefault` or whatever.

Reply:
Implemented with `Default` for the sake of simplicity :)